atomics/T1015/T1015.yaml

```diff
@@ -3,222 +3,46 @@ attack_technique: T1015
 display_name: Accessibility Features
 
 atomic_tests:
-- name: Attaches Command Prompt As Debugger To Process - osk
+- name: Attaches Command Prompt As Debugger To a Hashtable of target processes
   description: |
     This allows adversaries to execute the attached process
+    Attaches cmd.exe to osk.exe by default 
+    eg. $myArgs = @{"parent_list" = '"osk.exe", "sethc.exe", "utilman.exe", "magnify.exe", "narrator.exe", "DisplaySwitch.exe", "atbroker.exe"'}
+        Invoke-AtomicTest T1015 -InputArgs $myArgs
```

Comment:
Consider removing the reference to the execution framework (Invoke-AtomicRedTeam) from the test, as this may confuse users who are running this test based on the Markdown example.

Reply:
Done.